src/commands/apply.ts

```diff
@@ -185,7 +185,7 @@ async function applyManifests(
         }
         return null;
       }, 1),
-      filter((m): m is ManifestObject => m !== null),
+      filter((m): m is ManifestFileObject => m !== null),
```

Comment:
I'm not familiar with `is` but wouldn't the type guard throw an error here?

Reply:
no, `m is ManifestFileObject` narrows down the type to `ManifestFileObject` as long as the returned value is `true`.

Reply:
Oh I see. Gg